src/content/docs/en/guides/migrate-to-astro/from-gitbook.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Migrating from GitBook
+description: Tips for migrating an existing GitBook project to Astro
+type: migration
+stub: true
+framework: GitBook
+i18nReady: false
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[GitBook](https://gitbook.com) is a web-based platform for creating and publishing documentation and books in a collaborative manner, with version control integration and customizable features.
+
+
+## Key Similarities between GitBook and Astro
+
+GitBook and Astro share some similarities that will help you migrate your project:
+
+- Both Astro and GitBook support [Markdown](/en/guides/markdown-content/). You can migrate all your existing documentation utilizing GitBook's gitsync feature.
+
+- Both Astro and GitBook use some form of [file-based routing](/en/core-concepts/routing/). Using Astro's file structure for your existing content and when adding new pages should feel familiar.
+
+## Key Differences between GitBook and Astro
+
+When you migrate your GitBook docs to Astro, you will notice some important differences:
+
+- A GitBook site is edited using an online dashboard. In Astro, you will use a [code editor](/en/editor-setup/) and development environment to maintain your site. You can develop locally on your machine, or choose a cloud editor/development environment like StackBlitz, CodeSandbox or Gitpod.
+
+- GitBook stores your content in a database. In Astro, you will have individual files (typically Markdown or MDX) in your [project directory](/en/core-concepts/project-structure/) for each page's content. Or, you can choose to use a [CMS for your content](/en/guides/cms/), even your existing GitBook instance, and use Astro to fetch and present the data.
```

Comment:
> even your existing GitBook instance, and use Astro to fetch and present the data.

So, from what we explored before, looks like GitBook doesn't actually give you content in its API (currently), so I'm not sure about wording this like you can use GitBook and just fetch to Astro. True, you could use scraping but I think only a few peeps would try to go this route.

If that's really the case, I would just remove this bit.
```suggestion
- GitBook stores your content in a database. In Astro, you will have individual files (typically Markdown or MDX) in your [project directory](/en/core-concepts/project-structure/) for each page's content. Or, you can choose to use a [CMS for your content](/en/guides/cms/) and use Astro to fetch and present the data.
```